src/main/java/edu/hm/hafner/analysis/parser/GccParser.java

```diff
@@ -27,7 +26,7 @@ public class GccParser extends LookaheadParser {
     static final String LINKER_ERROR = "Linker error";
     private static final String GCC_WARNING_PATTERN = "^(?:\\s*(?:\\[.*\\]\\s*)?([^ ]*\\.[chpimxsola0-9]+):(?:(\\d*):"
             + "(?:\\d*:)*\\s*(?:(warning|error|note)\\s*:|\\s*(.*))|\\s*(undefined reference to.*))(.*)|.*ld:\\s*(.*-l("
-            + ".*)))$";
+            + ".*))|(?:In .+?:\\s*)?(cc1(?:plus)?):\\s*(warning|error|note):\\s*(.*))$";
```

Comment:
Hmm, this makes the parser more complex. Do you know if there are several different messages not picked up currently? I asked in JENKINS-73509, but maybe you have an answer already.